src/logmessage.cpp

```diff
@@ -125,7 +125,7 @@ namespace g3 {
 
    LogMessage::LogMessage(const std::string& file, const int line,
                           const std::string& function, const LEVELS& level)
-      : _timestamp(std::chrono::system_clock::now())
+      : _timestamp(std::chrono::high_resolution_clock::now())
```

Comment:
Maybe `g3::GetTimePoint()`?   I.e. so we stop breaking g3sinks after this change ;)